browser-test/src/support/ti_dashboard.ts

```diff
@@ -0,0 +1,54 @@
+import {Page} from 'playwright'
+
+export class TiDashboard {
+  public page!: Page
+
+  constructor(page: Page) {
+    this.page = page
+  }
+
+  async gotoTIDashboardPage(page: Page) {
+    await page.click('text="Trusted intermediary dashboard"')
+  }
+
+  async fillFormForNewClients(client: ClientInformation) {
+    await this.page.fill('label:has-text("Email Address")', client.emailAddress)
+    await this.page.fill('label:has-text("First Name")', client.firstName)
+    await this.page.fill('label:has-text("Middle Name")', client.middleName)
+    await this.page.fill('label:has-text("Last Name")', client.lastName)
+    await this.page.fill('label:has-text("Date Of Birth")', client.dobDate)
+    await this.page.click('text ="Add"')
+  }
+
+  async checkInnerTableForClientInformation(client: ClientInformation) {
+    const tableInnerText = await this.page.innerText('table')
+    expect(tableInnerText).toContain(this.convertToMMDDYYYY(client.dobDate))
+    expect(tableInnerText).toContain(client.emailAddress)
+    expect(tableInnerText).toContain(client.firstName)
+    expect(tableInnerText).toContain(client.lastName)
+  }
+  async checkInnterTableNotToConatain(client: ClientInformation) {
```

Comment:
Typo: `Innter` => `Inner`
Also name of the function should match the previous function. They both should have similar names with the only difference being "No" or "Not" or something like that in this function.